plugins/orchestrator/package.json

```diff
@@ -28,14 +28,16 @@
   ],
   "scripts": {
     "start": "backstage-cli package start",
-    "build": "backstage-cli package build",
+    "build": "yarn openapi:check && backstage-cli package build",
```

Comment:
I would suggest creating a dependency in turbo.json.

e.g. https://github.com/janus-idp/backstage-plugins/blob/main/plugins/servicenow-actions/turbo.json